Triangulation_3/include/CGAL/Regular_triangulation_3.h

```diff
@@ -166,12 +174,16 @@ namespace CGAL {
     using Tr_Base::find_conflicts;
     using Tr_Base::is_valid;
 
-    Regular_triangulation_3(const Gt & gt = Gt(), Lock_data_structure *lock_ds = NULL)
-      : Tr_Base(Tr_Base_Gt(gt), lock_ds), hidden_point_visitor(this)
+    Regular_triangulation_3()
+      : Tr_Base(Tr_Base_Gt(), NULL), hidden_point_visitor(this)
+    {}
+
+    Regular_triangulation_3(const Gt &, Lock_data_structure *lock_ds = NULL)
+      : Tr_Base(Tr_Base_Gt(), lock_ds), hidden_point_visitor(this)
```

Comment:
I do not understand that change. It the user has a custom `Gt` that carries state, you cannot ignore its value!

The left side of the patch was right: the `gt` object must be passed to the constructor of `Tr_Base_Gt`, and the constructor of `Tr_Base_Gt` must then pass `gt` to the constructor of its base class!